src/main/java/edu/rosehulman/covidtracer/service/PositiveCaseService.java

```diff
@@ -21,13 +25,18 @@
 	@Autowired
     PositiveCaseRepository repository;
 
-    public List<PositiveCase> getAllStudents(){
-        Pageable pageProperties = PageRequest.of(0, 10, Sort.by("id"));
+    public List<PositiveCase> getAllPositiveCases(Integer pageNum, Integer entriesPerPage, String sortBy){
```

Comment:
Are pageNum or entriesPerPage used in this?

Reply:
Hmmm. These are the values that make our GETS limited to 10 items per page. The values are received from the frontend and they determine how much data is sent at once. We might need to discuss as a team whether or not we want to use pages or not.

Reply:
I think pages are fine to go with for now. I think the users should decide weather they like that user experience.